tools/update-sha.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
```

Comment:
nit: add `set -eu` instruction in the beginning of script, this will make entire pipeline fail if at least one command returns non zero status and treat unset vars as error